revolv/templates/base/partials/dashboard_project.html

```diff
@@ -57,6 +57,30 @@ <h2 class="small-10 small-offset-1 columns">{{project.tagline}}</h2>
                 <div class="days-remaining-label">day{{project.days_left|pluralize}} left</div>
             </div>
         </div>
+        <div class="row project-stats">
+            <div class="project-stats-table-container small-12 large-10 large-offset-1 columns">
+                <div class="row project-stats-table-row">
+                    <div class="small-8 medium-6 columns">Kilowatt power output</div>
```

Comment:
Do we want to make these headings cms-able? In case there's a better description